_data/sites/web-components-dev.json

```diff
@@ -0,0 +1,7 @@
+{
+	"url": "https://webcomponents.dev/",
+	"name": "WebComponents.dev",
+	"description": "The Modern IDE for Web Platform Developers",
+	"twitter": "webcomp_dev",
+	"source_url": "https://github.com/zachleat/zachleat.com"
```

Comment:
I’m guessing this probably isn’t the right source URL? :D

Reply:
🤦‍♂